hollow/src/main/java/com/netflix/hollow/api/client/HollowBlobRetriever.java

```
@@ -32,18 +33,23 @@
      * @param desiredVersion the desired version
      * @return the snapshot for the state with an identifier equal to or less than the desired version
      */
-    public HollowBlob retrieveSnapshotBlob(long desiredVersion);
+    HollowBlob retrieveSnapshotBlob(long desiredVersion);
 
     /**
      * @param currentVersion the current version
      * @return a delta transition which can be applied to the currentVersion
      */
-    public HollowBlob retrieveDeltaBlob(long currentVersion);
+    HollowBlob retrieveDeltaBlob(long currentVersion);
 
     /**
      * @param currentVersion the current version
      * @return a reverse delta transition which can be applied to the currentVersion
      */
-    public HollowBlob retrieveReverseDeltaBlob(long currentVersion);
+    HollowBlob retrieveReverseDeltaBlob(long currentVersion);
 
+    /**
+     * @param desiredVersion the desired version
+     * @return the header for the state with an identifier equal to or less than the desired version
+     */
+    HollowBlobHeader retrieveHeaderBlob(long desiredVersion);
```

Add a default implementation here for backwards compatibility

---

This is deprecated but with only a single impl... 
I've added the default, just in case.

---

As you mentioned this is likely not used much, but just in case it would be better to throw an UnsupportedOperationException